cli/chain.go

```diff
@@ -193,6 +194,33 @@ var chainReadObjCmd = &cli.Command{
 	},
 }
 
+var chainDeleteObjCmd = &cli.Command{
```

Comment:
This is probably one of those commands which deserve the `--really-do-it` flag, and a warning